security/pkg/server/ca/server.go

```diff
@@ -34,7 +34,7 @@ import (
 	"istio.io/istio/security/pkg/pki/util"
 )
 
-var serverCaLog = log.RegisterScope("serverca", "Citadel server log")
```

Comment:
Change scope from serverca to ca to algin it with the [doc](https://istio.io/latest/docs/ops/diagnostic-tools/component-logging/#logging-scopes).

Reply:
maybe it is to distinguish ca client

Reply:
OK, changed back to serverca.